common/tier4_autoware_utils/src/vehicle/vehicle_state_checker.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "tier4_autoware_utils/vehicle/vehicle_state_checker.hpp"
+
+#include "tier4_autoware_utils/trajectory/trajectory.hpp"
+
+#include <string>
+
+namespace tier4_autoware_utils
+{
+VehicleStopChecker::VehicleStopChecker(rclcpp::Node * node)
+: clock_(node->get_clock()), logger_(node->get_logger())
+{
+  using std::placeholders::_1;
+
+  sub_odom_ = node->create_subscription<Odometry>(
+    "/localization/kinematic_state", rclcpp::QoS(1),
```

Comment:
Is there a way to change the topic used, or does it force the use of the specific topic?

Reply:
@satoshi-ota 
Could you add the document that describes what classes are provided and how to use them?

Reply:
Yes, it does. Only the localization output topic `/localization/kinematic_state` contains infomation whether the vehicle has stopped, so I think all nodes in autoware should use the topic.